ios/Codeforces Watcher/Application/components/FeedbackController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  FeedbackController.swift
+//  Codeforces Watcher
+//
+//  Created by Den Matyash on 5/6/20.
+//  Copyright © 2020 xorum.io. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+import common
+
+class FeedbackController: IFeedbackController {
+
+    private var KEY_CURRENT_SHOWING_ITEM: String {
```

Comment:
This strings keys can be reused in `common`, so consider moving them to `BaseFeedbackController`.

Reply:
i tried to do this but i haven't access to it

Reply:
Sure, it's private 😄Make it public and move to companion object.

Reply:
```
companion class {
    const val KEY_CURRENT_SHOWING_ITEM = "key_here"
}
```